src/assets/styles/20-tools/__manifest.scss

```diff
@@ -2,9 +2,9 @@
 // UI Tools
 // ----------------------------------
 // Design System 1.0 Alpha
-// 2019-11-11
+// 2019-01-10
```

Comment:
2020??

Reply:
Design system date is based on Jason's design work so we should leave the data as it was

Reply:
Ah okay, I'll update back to 2019-11-11 everywhere 👍🏻